CONTRIBUTING.md

```diff
@@ -50,57 +69,71 @@ influx query -o YOUR_ORG -t YOURAUTHTOKEN 'from(bucket:"example-bucket")
+If you have trouble including data in your report, please zip up your data directory and include a link to it in your bug report.
 
-Please note that issues are _not the place to file general support requests_ such as "how do I use collectd with InfluxDB?"
-Questions of this nature should be sent to the [InfluxData Community](https://community.influxdata.com/), not filed as issues.
+Note that issues are _not the place to file general support requests_ such as "How do I use `collectd` with InfluxDB?"
+Please submit requests for help to the [InfluxData Community](https://community.influxdata.com/) - don't report them as issues in the repo.
 
-## Feature requests
+## How to request a feature
 
-We really like to receive feature requests as it helps us prioritize our work.
-Please be clear about your requirements and goals, help us to understand what you would like to see added to InfluxD with examples and the reasons why it is important to you.
-If you find your feature request already exists as a Github issue please indicate your support for that feature by using the "thumbs up" reaction.
+We encourage you to submit feature requests as they help us prioritize our work.
 
-## Submitting a pull request
+In your feature request, please include the following:
+- Clear requirements and goals.
+- What you would like to see added to InfluxDB.
+- Examples.
+- Why the feature is important to you.
 
-To submit a pull request you should fork the InfluxDB repository, and make your change on a feature branch of your fork.
-Then generate a pull request from your branch against _master_ of the InfluxDB repository.
-Include in your pull request details of your change -- the why _and_ the how -- as well as the testing your performed.
-Also, be sure to run the test suite with your change in place.
-Changes that cause tests to fail cannot be merged.
+If you find your request already exists in a Github issue,
+please indicate your support for the existing issue by using the "thumbs up" reaction.
 
-There will usually be some back and forth as we finalize the change, but once that completes it may be merged.
+## Submitting a pull request
+
+## How to submit a pull request
 
 To assist in review for the PR, please add the following to your pull request comment:
 
-```md
-- [ ] CHANGELOG.md updated
-- [ ] Rebased/mergable
-- [ ] Tests pass
-- [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
-```
+1. If you haven't already, please [sign the CLA](#sign-the-cla).
+2. Fork the `influxdata/influxdb` repository.
+3. Checkout a feature branch in your fork.
+4. In your feature branch, make your change and run the test suite (given you won't be able to merge if tests fail).
+5. Commit your change.
+6. Generate a pull request (PR) from your branch against the **master** branch of `influxdata/influxdb`.
+7. Include the following in your PR description:
```

Comment:
Since we already have a PR template, we should just suggest here that they properly fill that template out.